packages/calcite-components/src/components/input-date-picker/input-date-picker.stories.ts

```diff
@@ -234,6 +241,60 @@ export const widthSetToBreakpoints_TestOnly = (): string =>
     html`<calcite-input-date-picker scale="{scale}" value="2020-12-12"></calcite-input-date-picker>`,
   );
 
+export const rangeWithValueAsDate = (): string => html`
+  <calcite-input-date-picker range open></calcite-input-date-picker>
+  <script>
+    const datePicker = document.querySelector("calcite-input-date-picker");
+    datePicker.valueAsDate = [new Date("2025-09-08"), new Date("2025-12-10")];
+  </script>
+`;
+
+rangeWithValueAsDate.parameters = {
+  chromatic: { delay: 2000 },
+};
+
+export const rangeWithValue = (): string => html`
+  <calcite-input-date-picker range open></calcite-input-date-picker>
+  <script>
+    const datePicker = document.querySelector("calcite-input-date-picker");
+    datePicker.value = ["2025-09-08", "2026-12-10"];
+  </script>
+`;
+
+rangeWithValue.parameters = {
+  chromatic: { delay: 2000 },
+};
+
+export const rangeWithMinBeforeCurrentDate = (): string => html`
```

Comment:
Prefer to keep them separate as they display different behavior in each case. 
When min is before currentDate, current day is viewed
When max is before currentDate, last valid day is  viewed since it is the closest day to the current day.